model/src/main/kotlin/FileFormat.kt

```diff
@@ -64,6 +64,16 @@ enum class FileFormat(val mapper: ObjectMapper, val fileExtension: String, varar
          * Return the [FileFormat] for the given [file], or `null` if there is none.
          */
         fun forFile(file: File): FileFormat = forExtension(file.extension)
+
+        /**
+         * Return a list of all files inside [directory] with known extensions that can be deserialized.
+         */
+        fun findDeserializableFiles(directory: File): List<File> {
```

Comment:
Would it make sense to name this e.g. `findFilesWithKnownExtension`
to not communicate that the returned files can actually be deserialized?

Reply:
Makes sense, will do.